test/tests/functional/pbs_sched_preempt_enforce_resumption.py

```diff
@@ -105,9 +105,10 @@ def test_suspended_job_ded_time_calendared(self):
         end = start + 120
         self.scheduler.add_dedicated_time(start=start, end=end)
 
+        temp += 180
         j1 = Job(TEST_USER)
         j1.set_attributes({ATTR_l + '.select': '1:ncpus=2',
-                           ATTR_l + '.walltime': 180})
+                           ATTR_l + '.walltime': temp-10})
```

Comment:
This is still a race.  If a slow machine takes more than 10s to submit this job (unlikely, but out testbed machines are really slow), you should make this start - int(time.time()) - 10.  This takes into account any time passed between when we create temp and when we set the walltime.